client/lib/site/utils.js

```diff
@@ -58,5 +58,29 @@ export default {
 		if ( site.options ) {
 			return site.options.default_post_format;
 		}
+	},
+
+	canUpdateFiles( site ) {
+		if ( site && ! site.hasMinimumJetpackVersion ) {
```

Comment:
I think we should check for `if ( ! site )` at the beginning of the method. So the following if doesn't fail if `site` is undefined.